### PR TITLE
cadence: update to 0.9.2

### PR DESCRIPTION
**Pull request by hervyqa**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)

Screenshot_20220620_123239

### CODE REVIEW

File: srcpkgs/cadence/template [hunk 1]
```diff
@@ -15,7 +14,8 @@ maintainer="lemmi <lemmi@nerd2nerd.org>"
 license="GPL-2.0-only"
 homepage="http://kxstudio.linuxaudio.org/Applications:Cadence"
```
Comment (paper42) [inline srcpkgs/cadence/template:15]:
```suggestion
homepage="https://kx.studio/Applications:Cadence"
```

**Review by paper42 — COMMENTED** [review]
@lemmi

File: srcpkgs/cadence/template (hunk 1 above)
Comment (hervyqa) [reply srcpkgs/cadence/template]: Thanks.
done..

**Comment by lemmi** [comment]
Thanks :)

**Pull request closed without merging**